pkg/agent/apiserver/apiserver.go

```diff
@@ -55,6 +55,7 @@ const Name = "antrea-agent-api"
 var (
 	scheme = runtime.NewScheme()
 	codecs = serializer.NewCodecFactory(scheme)
+	// TokenPath defines the path of token file
```

Comment:
not a good comment IMO, it should explain what the token is for? Also, why the change, I thought you didn't enable that revive rule?

Reply:
these are some previous changes I made for golint, I kept them since I feel most of them are reasonable warnings and fix. I will remove this one and double check if there is any other unnecessary changes